lua/telescope/make_entry.lua

```diff
@@ -324,6 +324,10 @@ function make_entry.gen_from_quickfix(opts)
 
     local line_info = { table.concat({ entry.lnum, entry.col }, ":"), "TelescopeResultsLineNr" }
 
+    if opts.trim_text then
```

Comment:
We need documentation for this option:.
Docs is autogenerated from this file: https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/builtin/init.lua

`make_entry.gen_from_quickfix` is used for:
- `lsp_references`
- `lsp_definitions`
- `lsp_type_definitions`
- `lsp_implementations?
- `quickfix`
- `loclist`
- `tagstack`
- `jumplist`